pages/instructorGradingJob/instructorGradingJob.sql

```diff
@@ -24,5 +24,5 @@ FROM
     LEFT JOIN users AS u ON (u.user_id = s.auth_user_id)
 WHERE
     gj.id = $job_id
-    AND gj.grading_method = 'External'
+    AND gj.grading_method_external = True
```

Comment:
So this is one area that it is nice to have the grading_method_external on the grading_job. 

If a graded submission has a grading job, I believe we could just look for these boolean flags in the `grading_job` table. 

Do you believe that it is necessary to put the grading method booleans in the submission as well?

Reply:
It's just tied to the submission in so many other places in the code--I think the frontend queries base it off of the submission columns, rather than the grading job columns, in many places. I think it was tied to the submission earlier because of that--the data is easily at hand without doing table joins.